### DISCUSSION TITLE
Lutris has encountered an error

### DISCUSSION THREAD

**Discussion by NarakuITA** [lutris/lutris#5636] [General]
I don't understand, but when I try to install EA App or Epic Games Store once the launchers are downloaded the installation fails and returns this error:

```
Il comando è terminato con il codice 

Traceback (most recent call last):

  File "/usr/lib/python3.12/site-packages/lutris/exception_backstops.py", line 76, in error_wrapper
    return handler(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3.12/site-packages/lutris/util/jobs.py", line 127, in wrapper
    repeat = func(*a, **kw)
             ^^^^^^^^^^^^^^

  File "/usr/lib/python3.12/site-packages/lutris/installer/commands.py", line 444, in _monitor_task
    raise ScriptingError(_("Command exited with code %s") % command.return_code)

lutris.installer.errors.ScriptingError: Il comando è terminato con il codice

Lutris log:
[INFO:2024-10-06 11:34:24,095:application]: Starting Lutris 0.5.17
[INFO:2024-10-06 11:34:24,221:startup]: "card0" is NVIDIA GeForce RTX 3060 (10de:2504 19da:2630 nvidia) Driver 560.35.03
[INFO:2024-10-06 11:36:38,812:file_box]: Download completed
[INFO:2024-10-06 11:37:01,840:file_box]: Download completed
[INFO:2024-10-06 11:37:01,841:files_box]: All files available
[INFO:2024-10-06 11:37:01,841:installerwindow]: All files are available, continuing install
[INFO:2024-10-06 11:37:01,843:installerwindow]: Launching installer commands
[INFO:2024-10-06 11:37:01,924:wine]: Creating a win64 prefix in /home/andrea/Games/epic-games-store
[INFO:2024-10-06 11:37:01,924:wine]: Winepath: /home/andrea/.local/share/lutris/runners/wine/wine-ge-8-26-x86_64/bin/wine
[INFO:2024-10-06 11:37:08,618:wine]: win64 Prefix created in /home/andrea/Games/epic-games-store
[WARNING:2024-10-06 11:37:08,877:monitored_command]: No file /tmp/lutris-32c4e95d-d9b7-4c7e-a557-c94a47ef949d
[ERROR:2024-10-06 11:37:09,803:errors]: Il comando è terminato con il codice 
[ERROR:2024-10-06 11:37:09,803:exception_backstops]: Error handling timeout function: Il comando è terminato con il codice 
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/lutris/exception_backstops.py", line 76, in error_wrapper
    return handler(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/lutris/util/jobs.py", line 127, in wrapper
    repeat = func(*a, **kw)
             ^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/lutris/installer/commands.py", line 444, in _monitor_task
    raise ScriptingError(_("Command exited with code %s") % command.return_code)
lutris.installer.errors.ScriptingError: Il comando è terminato con il codice
```

What should I do to fix this? I am using the latest version and am using updated Nabara OS.

**Comment by seminewb** [comment]
Similar issue with Ubisoft Installer posting here for bug fixing purposes. after failed Ghostwire Tokyo install on both Heroic and Lutris.

My OS is Linux Mint 22, KERNEL 6.8.0-51-GENERIC ARCH
DESKTOP CINNAMON V: 6.2.9 tk: GTK v: 3.24.41 wm: Muffin v: 6.2.0 vt: 7 dm: LightDM v: 1.30.0

Command exited with code 23296

Traceback (most recent call last):

  File "/app/lib/python3.11/site-packages/lutris/exception_backstops.py", line 79, in error_wrapper
    return handler(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^

  File "/app/lib/python3.11/site-packages/lutris/util/jobs.py", line 127, in wrapper
    repeat = func(*a, **kw)
             ^^^^^^^^^^^^^^

  File "/app/lib/python3.11/site-packages/lutris/installer/commands.py", line 447, in _monitor_task
    raise ScriptingError(_("Command exited with code %s") % command.return_code)

lutris.installer.errors.ScriptingError: Command exited with code 23296

Lutris log:
[INFO:2024-12-29 23:29:53,885:application]: Starting Lutris 0.5.18
[INFO:2024-12-29 23:29:53,926:startup]: "card1" is AMD Radeon RX 6650 XT (1002:73ef 1043:05e3 amdgpu) Driver 24.3.1
[INFO:2024-12-29 23:30:36,153:interpreter]: Runner wine needs to be installed
[INFO:2024-12-30 00:04:02,954:file_box]: Download completed
[INFO:2024-12-30 00:04:02,954:files_box]: All files available
[INFO:2024-12-30 00:04:02,954:installerwindow]: All files are available, continuing install
[INFO:2024-12-30 00:04:02,955:installerwindow]: Launching installer commands
[INFO:2024-12-30 00:04:03,026:wine]: Creating a win64 prefix in /home/paul/Games/ubisoft-connect
[INFO:2024-12-30 00:04:03,026:wine]: Winepath: /home/paul/.var/app/net.lutris.Lutris/data/lutris/runners/wine/wine-ge-8-26-x86_64/bin/wine
[INFO:2024-12-30 00:04:11,842:wine]: win64 Prefix created in /home/paul/Games/ubisoft-connect
[ERROR:2024-12-30 00:05:38,380:errors]: Command exited with code 23296
[ERROR:2024-12-30 00:05:38,380:exception_backstops]: Error handling timeout function: Command exited with code 23296
Traceback (most recent call last):
  File "/app/lib/python3.11/site-packages/lutris/exception_backstops.py", line 79, in error_wrapper
    return handler(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/util/jobs.py", line 127, in wrapper
    repeat = func(*a, **kw)
             ^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/installer/commands.py", line 447, in _monitor_task
    raise ScriptingError(_("Command exited with code %s") % command.return_code)
lutris.installer.errors.ScriptingError: Command exited with code 23296
